cli/sawtooth_cli/main.py

```diff
@@ -83,6 +84,18 @@ def create_parent_parser(prog_name):
         action='count',
         help='enable more verbose output')
 
+    try:
+        version = pkg_resources.get_distribution('sawtooth-cli').version
+    except pkg_resources.DistributionNotFound:
+        print("please install sawtooth-cli with setup.py")
+        return
+
+    parent_parser.add_argument(
+        '-V', '--version',
+        action='version',
```

Comment:
No, I want to use ```parser.add_argument('--version', action='version', version='...')``` on https://docs.python.org/3.5/library/argparse.html, The version= keyword argument prints version information and exits when invoke.